docs/src/pages/pt/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Introdução
```

Comment:
```suggestion
title: Introdução
lang: "pt"
```
Could you add a language code so the html element automatically gets the right lang attribute on it?

Reply:
Should also be `pt_BR`

Reply:
I also believe this is Portuguese from Brazil, so the language tag should be `pt_BR`. Is that right?

Reply:
This is indeed portuguese from brazil, but i'm not sure about the pt_BR flag

Reply:
It is. Check it out: 
https://en.wikipedia.org/wiki/Brazilian_Portuguese#Language_codes

Reply:
I can certainly use the pt-br flag for the lang, but for the file name, I tried to follow what was already there (using the language only convention and not the language-region).

Reply:
maybe that's something that's changing? I think Fred added support for the extra specifier in the dropdown.. for example this PR doesn't follow the rule: https://github.com/snowpackjs/astro/pull/980

Reply:
Will update it then with the proper region identifier.

Reply:
You guys are the best!